.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a helm chart update (eg: CRD changes)
+- [ ] E2e not required (eg: Updates in test cases)
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes 
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
+
```

Comment:
Can we also add a check point not to break exiting UTs and ITs
Can we also add static code analysis like sonarcube or staticCheck

Reply:
they will be part of PR checks